products/citrix-apps-desktops.md

```diff
@@ -103,6 +122,6 @@ Citrix Virtual Apps and Desktops was previously known as XenApp and XenDesktop,
 There are two release types for on-premises deployments, Current Releases and Long Term Service Releases (LTSR).
 
 * Current Releases will reach end of active support 6 months after release and end of security support 18 months after release.
-* Long Term Service Releases will reach end of active and security support 5 years after release. Incremental updates are delivered in the form of Cumulative Updates (CU). LTSR is also eligible for paid extended support up to 5 years after active and security support ends and is not reflected above.
+* Long Term Service Releases will reach end of active and security support 5 years after release. Incremental updates are delivered in the form of Cumulative Updates (CU). LTSR is also eligible for paid extended support up to 5 years after active and security support ends.
```

Comment:
@lryanuk, do you think that would be valuable to add a link to https://support.citrix.com/article/CTX459999/extended-support-customer-guide ?